game-core/src/main/java/games/strategy/triplea/ui/MapPanel.java

```
@@ -447,27 +443,27 @@ void setGameData(final GameData data) {
     gameData.addTerritoryListener(territoryListener);
     gameData.addDataChangeListener(dataChangeListener);
     clearPendingDrawOperations();
-    executor.execute(() -> tileManager.resetTiles(gameData, uiContext.getMapData()));
+    executor.submit(() -> tileManager.resetTiles(gameData, uiContext.getMapData()));
```

Same here, about swallowed errors